harness/src/main/java/org/jboss/pvt/harness/utils/ClassVersionInspector.java

```diff
@@ -101,4 +97,43 @@ public static boolean checkJarVersion (String filename, ClassVersion version) th
 
         return true;
     }
+
+    /**
+     * Support version scope
+     * @param filename
+     * @param minVersion
+     * @param maxVersion
+     * @return
+     * @throws PVTException
+     */
+    public static boolean checkJarVersion (String filename, ClassVersion minVersion, ClassVersion maxVersion) throws PVTException
+    {
```

Comment:
If this is for PRODCI-34 then that is version format check - nothing to do with version of the classes (which is jdkcompatiblecheck) ? Can this be removed and moved into a separate PR please